source/agora/consensus/data/UTXO.d

```diff
@@ -141,6 +265,24 @@ public class TestUTXOSet
         }
         return false;
     }
+
+    /// Ditto
+    protected override void remove (Hash utxo) @safe
+    {
+        this.storage.remove(utxo);
+    }
+
+    /// Ditto
+    protected override void add (Hash utxo, UTXO value) @safe
+    {
+        this.storage[utxo] = value;
+    }
+
+    /// Ditto
+    protected override bool find (Hash utxo, UTXO value) nothrow @safe
+    {
+        return this.peekUTXO(utxo, value);
+    }
```

Comment:
Remove `find` and just expose `peekUTXO` in both cases.

Reply:
I used `peekUTXO` for `TestUTXOSet` since it doesn't have its own `find` and `peekUTXO` is doing the same thing. However, `UTXOSet` doesn't have the `peekUTXO` method.

As you suggested, it may the best to remove the `getUTXO` in the base class ?

Reply:
I think the real UTXO should also have `peekUTXO`

Reply:
What do you think about renaming the current `find` overload to `peekUTXO`? It is calling this in `UTXODB`.

https://github.com/bpfkorea/agora/blob/7a7a5b186443de3b2bc676ff08247f67bdb98551/source/agora/consensus/state/UTXOSet.d#L263-L276

Reply:
Yes that was the suggestion :)